src/apps/api/views/competitions.py

```diff
@@ -201,6 +209,22 @@ def email_all_participants(self, request, pk):
         batch_send_email.apply_async((comp.pk, content))
         return Response({}, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['GET',],)
+    def get_csv(self, request, pk,):
+        comp = Competition.objects.get(pk=pk)
+        with SpooledTemporaryFile() as tmp:
```

Comment:
Awesome use of this!! I haven't seen `SpooledTemporaryFile` yet, very cool:
https://www.kite.com/python/docs/tempfile.SpooledTemporaryFile